jcl/src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -79,7 +79,15 @@
 
 	// Instance variables
 	private long threadRef;									// Used by the VM
-	long stackSize = 0;
+    long stackSize = 0;
```

Comment:
The indenting is off in the new section. Using spaces instead of tabs like the surrounding code.